src/Effects.js

```diff
@@ -600,3 +633,56 @@ function drawSpiral(ctx) {
   ctx.fill();
   ctx.restore();
 }
+function drawSparkle(ctx,color) {
+  ctx.save();
+  ctx.beginPath();
+  ctx.moveTo(0,0);
+  ctx.lineTo(12,0);
+  ctx.lineTo(12,12);
+  ctx.lineTo(0,12);
+  ctx.closePath();
+  ctx.clip();
+  ctx.translate(0,0);
+  ctx.translate(0,0);
+  ctx.scale(0.22099447513812157,0.22099447513812157);
+  ctx.translate(0,0);
+  ctx.strokeStyle = 'rgba(0,0,0,0)';
+  ctx.lineCap = 'butt';
+  ctx.lineJoin = 'miter';
+  ctx.miterLimit = 4;
+  ctx.save();
+  ctx.fillStyle = color;
+  ctx.beginPath();
+  ctx.moveTo(54.3,27.2);
+  ctx.bezierCurveTo(30.699999999999996,29.099999999999998,29.099999999999998,30.7,27.199999999999996,54.3);
+  ctx.bezierCurveTo(25.2,30.7,23.6,29.1,0,27.2);
+  ctx.bezierCurveTo(23.6,25.2,25.2,23.6,27.2,0);
+  ctx.bezierCurveTo(29.1,23.6,30.7,25.2,54.3,27.2);
+  ctx.closePath();
+  ctx.fill();
+  ctx.stroke();
+  ctx.restore();
+  ctx.save();
+  ctx.fillStyle = color;
+  ctx.beginPath();
```

Comment:
Each of these sparkles actually has a second, hidden border path that I think we can skip drawing.  Fixing:

![image](https://user-images.githubusercontent.com/413693/48580191-641a7880-e8d3-11e8-9e7c-0336632284bf.png)

Reply:
Fixed in commit 32b81cd7566e7ac80416ab3b7477cd678228e437.